test/local/senders/email.js

```diff
@@ -779,6 +779,26 @@ describe(
       }
     )
 
+    it(
+      'sends request to fxa-email-server when the email pattern is right',
+      function() {
+        var message = {
+          email: 'emailservice.foo@restmail.net',
+          flowId: 'wibble',
+          subject: 'subject',
+          template: 'verifyLoginEmail',
+          uid: 'foo'
+        }
+
+        var stub = sinon.stub(mailer.emailService, 'sendMail')
+        mailer.send(message)
+          .then(
+            response => assert(stub.called),
```

Comment:
This is great.

But because I'm a bit paranoid when it comes to assertions, I think we should also assert that the other mailer isn't called in this case. And in the test where we assert that one is called, we should assert this one isn't called. And I'd also assert that the arguments here are still correct, even though the same bit of code is responsible for passing them in.